docs/dev_coding_standard.rst

```diff
@@ -0,0 +1,13 @@
+
+===============
+Coding Standard
+===============
+
+The coding standard for libCellML follows that specified in the `google c++ style guide <http://google-styleguide.googlecode.com/svn/trunk/cppguide.html>`_.  But it does not have to be followed to the letter, `other people <https://www.linkedin.com/pulse/20140503193653-3046051-why-google-style-guide-for-c-is-a-deal-breaker>`_ have not so positive opinions about the quality of the google style guide.  
+
+In essence we seek code that looks good, is easy to read and has great documentation.  We don't want to spend time discussing the minutiae of the coding style.
```

Comment:
Good!

Perhaps add a comment about try to be consistent with the style of existing code you add to?